src/ekore/anomalous_dimensions/unpolarized/time_like/as1.py

```diff
@@ -0,0 +1,129 @@
+"""The following are the unpolarized time-like leading-order (LO) Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+from eko import constants
+
+
+
+@nb.njit(cache=True)
+def gamma_qq(N, s1):
+    """Computes the LO quark-quark anomalous dimension.
+    Implements Eqn. (B.3) from hep-ph/0604160.
+
+    Parameters
+    ----------
+    N : complex
+        Mellin moment 
+    s1 : complex
+        Harmonic sum $S_{1}$ 
+
+    Returns
+    -------
+    gamma_qq : complex
+        LO quark-quark anomalous dimension $\gamma_{qq}^{(0)}(N)$ 
+
+    """
+    return constants.CF * (-3.0 + (4.0 * s1) - 2.0 / (N * (N + 1.0)))
```

Comment:
this is the same as https://github.com/NNPDF/eko/blob/7a74ddbfdfff5b485de3aec20d7f1fbcd33b6c29/src/ekore/anomalous_dimensions/unpolarized/space_like/as1.py#L10 so you should not reimplement, but just redirect (same of course for all the others); only `gamma_singlet` changes if I understood correctly (so that one you keep)

Reply:
well actually, only the 2 functions, namely gamma_ns and gamma_gg are main the same. gamma_gq and gamma_qg do differ by factors of nf. I think it would be better and clearer if we have the _ns and _gg functions clearly specified and not redirect to the space-like folder because it provides a cleaner and clearer separation between TL and SL and also allows for proper citation of references with functions.

Reply:
Even fine ... are there relations beyond LO between TL and SL?

Reply:
no, its only for the LO afaik, for NLO and NNLO, they vary, so there will be different functions there anyway

Reply:
Then I'm also fine with duplicating two lines ... just remember to add a unit test that actually checks that